### PR TITLE
Fix trt Test

### PR DESCRIPTION
**Pull request by codeislife99**
Fix TRT Test

### CODE REVIEW

File: tests/python/contrib/test_tensorrt.py [hunk 1]
```diff
@@ -1058,7 +1058,7 @@ def test_tensorrt_dynamic_batch():
         mod = tvm.IRModule()
         mod["main"] = f
         if use_trt:
-            mod = relay.tensorrt.EnableTrt(mod)
+            mod = tensorrt.partition_for_tensorrt(mod, params)
```
Comment (trevor-m) [inline tests/python/contrib/test_tensorrt.py:1061]: I don't think this model has params

**Review by trevor-m — CHANGES_REQUESTED** [review]

File: tests/python/contrib/test_tensorrt.py [hunk 2]
```diff
@@ -1050,26 +1050,26 @@ def test_tensorrt_dynamic_batch():
...
-            mod = relay.tensorrt.EnableTrt(mod)
+            mod, _ = tensorrt.partition_for_tensorrt(mod)
 
         if not skip_runtime_test():
             with relay.build_config(opt_level=3):
                 relay_exec = relay.create_executor("vm", mod=mod, ctx=tvm.cpu(0), target="llvm")
 
             for i, batch_size in enumerate(batches_to_test):
-                result_dict[(i, use_trt)] = relay_exec.evaluate()(x_data[:batch_size, ...])
+                result_dict[use_trt] = relay_exec.evaluate()(x_data[:batch_size, ...])
```
Comment (trevor-m) [inline tests/python/contrib/test_tensorrt.py:1068]: I think we still need to include `i` in the result dict

**Review by trevor-m — APPROVED** [review]
Thanks!

**Review by anijain2305 — APPROVED** [review]

**Comment by anijain2305** [comment]
Thanks @codeislife99 @trevor-m This is merged

**Pull request merged**